### ISSUE TITLE
Add support for SonarQube preview report

### ISSUE THREAD

**Issue by congtq** [tomasbjerre/violations-lib#66]
Please add support for SonarQube preview report.
SonarQube has GitLab plugin but it seem it does not work in SonarQube Community edition.

Here is format of sonar report:
File name pattern: `.*/target/sonar/sonar-report\.json`  
File content example (in pretty format):
```json
{
  "version":"6.7.4.38452",
  "issues": [{
    "key": "AfhgjhgUhjhkhkOqm3c-",
    "component": "com.example:myapp:src/main/java/com/example/myapp/App.java",
    "line": 28,
    "startLine": 28,
    "endLine": 28,
    "message": "Complete the task associated to this TODO comment.",
    "severity": "INFO",
    "rule": "squid:S1135",
    "status": "OPEN",
    "isNew": false,
    "creationDate": "2019-01-01T11:12:13+0900"
  }, {
	...
  }],
  ...
}
```

**Comment by tomasbjerre** [comment]
Should be no problem. How do you generate this report file? What command do you execute?

**Comment by congtq** [comment]
I use follow command to get `sonar-report.json` with sonar maven plugin
```
mvn sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.report.export.path=sonar-report.json
```

**Comment by tomasbjerre** [comment]
Released in Violation Comments to Gitlab Plugin 2.22.

**Comment by tomasbjerre** [comment]
I just saw this issue: https://jira.sonarsource.com/browse/SONAR-11670

So be aware, this report file will be removed when upgrading to later version of SonarQube.
